hazelcast/src/test/java/com/hazelcast/internal/metrics/impl/MetricsRegistryImplTest.java

```diff
@@ -112,6 +113,17 @@ public long get(Object obj) throws Exception {
     }
 
     @Test
+    public void renders_NA_when_requested_stat_not_available() throws Exception {
+        String requestedStatName = MetricsRegistryImplTest.this.getClass().getName();
```

Comment:
Actually it can be one of the stats [here](https://github.com/hazelcast/hazelcast/blob/master/hazelcast-client/src/main/java/com/hazelcast/client/impl/statistics/Statistics.java#L342). From this perspective, it is a known stat name but we don't have a method to read it.